joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -1384,6 +1384,11 @@ TEST_P(TrajectoryControllerTestParameterized, invalid_message)
   traj_msg = good_traj_msg;
   traj_msg.points.push_back(traj_msg.points.front());
   EXPECT_FALSE(traj_controller_->validate_trajectory_msg(traj_msg));
+
+  // End time in the past
+  traj_msg = good_traj_msg;
+  traj_msg.header.stamp = rclcpp::Time(1);
+  EXPECT_FALSE(traj_controller_->validate_trajectory_msg(traj_msg));
```

Comment:
Is this enough to test? 
Maybe we should add two tests:
- One with start_time + time_from_start < now() -> fail
- One with start_time + time_from_start > now() -> success

Reply:
Good point, I have added one such successful case as well.